bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithNavigationControlXML.java

```diff
@@ -297,8 +300,10 @@ public void update() throws IOException, ReceivedMessageParseException {
 
             totalWaitingTime += YamahaReceiverBindingConstants.MENU_RETRY_DELAY;
             if (totalWaitingTime > YamahaReceiverBindingConstants.MENU_MAX_WAITING_TIME) {
-                throw new IOException(
-                        "Menu still not ready after " + YamahaReceiverBindingConstants.MENU_MAX_WAITING_TIME + "ms");
+                logger.info("Menu still not ready after " + YamahaReceiverBindingConstants.MENU_MAX_WAITING_TIME + "ms. The menu state will be out of sync.");
+                // ToDo: this needs to redesigned to allow for some sort of async update
```

Comment:
There is a TODO, what should be done and can it be completed before merging the PR?

Reply:
I put that ToDo for another time. I am not using menus myself and not sure how it works, but when you look at the code it would require a rework to avoid delaying and a look to fetch updates. Too big refactor for now. 

With this PR I am trying to fix the rather critical issue introduced last time.